libc/calls/calls.mk

```diff
@@ -51,7 +51,8 @@ LIBC_CALLS_A_DIRECTDEPS =				\
 	LIBC_STR					\
 	LIBC_STUBS					\
 	LIBC_SYSV_CALLS					\
-	LIBC_SYSV
+	LIBC_SYSV					\
+	LIBC_VGA
```

Comment:
Please remove this dependency. `LIBC_CALLS` should not need to depend on `LIBC_VGA` because you're using weak linking.

Reply:
Thank you!  Let me try again without this dependency.